src/Nncase.Core/Utilities/ReplaceUtility.cs

```diff
@@ -91,4 +96,22 @@ public static Call ReplaceCallParams(Expr target, IReadOnlyList<Expr> oldParams,
     {
         return new Call(target, ImmutableArray.CreateRange(ReplaceItems(oldParams, pairs)));
     }
+
+    /// <summary>
+    /// replace the call params with parameter info.
+    /// </summary>
+    /// <param name="call">call.</param>
+    /// <param name="pairs">the param info pair.</param>
+    /// <returns>new call.</returns>
+    public static Call ReplaceCallParams(Call call, params (Expr, Expr)[] pairs) =>
+        ReplaceCallParams(call.Target, call.Parameters, pairs);
```

Comment:
Do not use the `call.Target` or `call.Parameters` in GetReplace. refer:https://github.com/kendryte/nncase/pull/789#issuecomment-1408046184